siriuspy/siriuspy/pwrsupply/pructrl/prucontroller.py

```diff
@@ -795,3 +807,10 @@ def _dict2list_vargroups(groups_dict):
         # create list of variable ids
         groups_list = [groups_dict[gid] for gid in group_ids]
         return groups_list
+
+    def _read_parameter_values(self, device_ids=None):
```

Comment:
@anacso17 , I think you should change the method name to `_bsmp_read_parameter_values`. All primary class methods that do bsmp comunication have names starting with prefix `_bsmp`. See

https://github.com/lnls-sirius/dev-packages/blob/011c4f6d471b0c0ada3b24d4e7804b58bf1dd622/siriuspy/siriuspy/pwrsupply/pructrl/prucontroller.py#L32

Reply:
Really forgot that detail, fixed! I also changed the order of the methods to group this one with the other `_bsmp` methods